tests/attention/test_trtllm_gen_block_sparse_decode.py

```
@@ -0,0 +1,597 @@
+        q,
+        q_scale,
+        ref_q,
+        kv_cache,
+        k_scale,
+        v_scale,
+        ref_kv_cache,
+        page_table,
+        head_tables,
+        sparse_lens,
+        selected_positions,
+        seq_lens,
```

Ruff (0.15.21)

[warning] Unpacked variable `seq_lens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)